cnd/src/config/validation.rs

```diff
@@ -25,6 +26,19 @@ pub enum NetworkValidationResult<T> {
     }
 }
 
+pub async fn validate_connected_network<T: std::cmp::PartialEq, U, F: Fn(U) -> Box<dyn Future<Item=NetworkValidationResult<T>, Error=anyhow::Error>>> (
```

Comment:
Are you running `rustfmt` in your editor?  It seems strange to see this `=` with no spaces around it?